README.rst

```diff
@@ -23,6 +23,17 @@ Please don't commit changes with reStructuredText syntax errors that cause PEP
 generation to fail, or result in major rendering defects relative to what you
 intend.
 
+ReadTheDocs preview
+-------------------
+
+For every PR, we automatically create a preview of the rendered PEPs using
+ReadTheDocs. You can find it in the branch action box at the bottom of the
+PR page:
```

Comment:
```suggestion
`ReadTheDocs <https://readthedocs.org/>`_.
You can find it in the merge box at the bottom of the PR page:
```

* Add link for RTD
* Rename "branch action box" to "merge box", since that's what [GitHub's docs appear to call it](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-stage-of-a-pull-request#marking-a-pull-request-as-ready-for-review), what I generally hear others call it, and what more specifically describes its primary function.

Reply:
I wasn't sure what to call, I got "branch action" from the CSS class name for the thing :D. "Merge box" is better.

Reply:
Ah, interesting! Was wondering where that came from...